src/MQTT433gateway.ino

```diff
@@ -66,7 +66,7 @@ boolean rawMode = false;
 String otaURL = "";
 
 void setup() {
-  Serial.begin(115200,SERIAL_8N1,SERIAL_TX_ONLY); 
+  Serial.begin(115200,SERIAL_8N1,SERIAL_TX_ONLY);
```

Comment:
Please also fix the missing spaces after the comma. (maybe better in a separate commit)